src/Jackett.Common/Indexers/EpubLibre.cs

```diff
@@ -0,0 +1,183 @@
+                        var languageId = qTooltip.QuerySelector("div.pull-right").GetAttribute("class").Split('_')[1];
+                        title += $" [{_languages[languageId]}] [epub]";
+                        var qDesc = qTooltip.QuerySelectorAll("div.row-fluid > div");
+                        var description = $"Rev: {qDesc[0].TextContent} Páginas: {qDesc[1].TextContent} Puntación: {qDesc[2].TextContent} Likes: {qDesc[3].TextContent}";
+
+                        // publish date is not available in the torrent list, but we add a relative date so we can sort
+                        lastPublishDate = lastPublishDate.AddMinutes(-1);
+
+                        var release = new ReleaseInfo
+                        {
+                            Title = title,
+                            Comments = comments,
+                            Link = comments,
+                            Guid = comments,
+                            PublishDate = lastPublishDate,
+                            BannerUrl = banner,
+                            Description = description,
+                            Category = new List<int> {TorznabCatType.BooksEbook.ID},
+                            Size = 5242880, // 5 MB
+                            Seeders = 1,
+                            Peers = 2,
+                            MinimumRatio = 1,
+                            MinimumSeedTime = 172800, // 48 hours
+                            DownloadVolumeFactor = 0,
+                            UploadVolumeFactor = 1
+                        };
+                        releases.Add(release);
+                    }
+
+                    if (rows.Length < MaxItemsPerPage)
+                        break; // this is the last page
+                }
+                catch (Exception ex)
+                {
+                    OnParseError(result.Content, ex);
+                }
+            }
+
+            return releases;
+        }
+
+        public override async Task<byte[]> Download(Uri link)
+        {
+            var result = await RequestStringWithCookies(link.AbsoluteUri);
+            try {
+                var parser = new HtmlParser();
+                var doc = parser.ParseDocument(result.Content);
+                var magnetLink = doc.QuerySelector("a[id=en_desc]").GetAttribute("href");
+                return Encoding.UTF8.GetBytes(magnetLink);
+            }
+            catch (Exception ex)
+            {
+                OnParseError(result.Content, ex);
+            }
+            return null;
+        }
+
+        // TODO: merge this method with query.MatchQueryStringAND
+        private static bool CheckTitleMatchWords(string queryStr, string title)
+        {
+            // this code split the words, remove words with 2 letters or less, remove accents and lowercase
+            var queryMatches = Regex.Matches(queryStr, @"\b[\w']*\b");
```

Comment:
Using Regex.Split might be better for this? Since it's the same Regex string for both, reusing the Regex through the following code.
```
var words = new Regex(@"\b[\w']+\b");
var queryWords = words.Split(queryStr)
...
var titleWords = words.Split(title);
```

Also, see if any of the options may help speed up this regex: https://docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regexoptions